modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectLayoutLocalServiceTest.java

```diff
@@ -374,8 +382,8 @@ public void testAddObjectLayout() throws Exception {
 
 		_assertObjectLayout(objectLayout);
 
-		_objectLayoutLocalService.deleteObjectLayout(
-			objectLayout.getObjectLayoutId());
+		_objectDefinitionLocalService.deleteObjectDefinition(
+			_objectDefinition.getObjectDefinitionId());
```

Comment:
You don't need to delete this variable because of this:
```
@DeleteAfterTestRun
private ObjectDefinition _objectDefinition;
```

Reply:
Could you maintain the deletion of the ObjectLayout please?

Reply:
Makes sense, thanks